### PR TITLE
week1 homework

### PR DESCRIPTION
**Pull request by husenjanobul**

### CODE REVIEW

File: Week1/homework/style.css [hunk 1]
```diff
@@ -1 +1,53 @@
...
+    text-shadow: 1px 1px 5px rgb(6, 6, 46);
+    margin: 2%;
+}
+ul{
+    list-style: none;
+    display: flex;
+    flex-wrap: wrap;
+}
+ul li{
+    box-shadow: 1px 1px 5px black;
```
Comment (wilgert) [inline Week1/homework/style.css]: Your page looks pretty nice. Only one thing. On wide screens the images are stretched horizontally:
Screenshot 2019-08-21 at 21 00 56


Please find a way to solve that.

File: Week1/homework/app.js [hunk 2]
```diff
@@ -4,8 +4,117 @@
...
+    baby_minds: {
+      title: 'Baby Minds',
+      language: 'English',
+      author: 'Linda Acredolo Susan Goodwin',
+      img: 'https://s.s-bol.com/imgbase0/imagebase3/regular/FC/5/3/5/8/1001004001348535.jpg',
+    },
+  };
+
+  function bookList() {
+    // Create ul element
```
[the remark this answers is not in the document]
Comment (husenjanobul) [reply Week1/homework/app.js]: Thanks your feedback . 
I’ll improve it .

File: Week1/homework/style.css (hunk 1 above)
Comment (husenjanobul) [reply Week1/homework/style.css]: Thank you,
I’ll fix it .

File: Week1/homework/app.js [hunk 3]
```diff
@@ -4,8 +4,117 @@
...
+    const bookUl = document.createElement('ul');
+    for (let i = 0; i < bookTitles.length; i++) {
+      // Create li teg
+      const bookLi = document.createElement('li');
+      // Div Of  Book's Image
+      const imgDiv = document.createElement('div');
+      imgDiv.setAttribute('id', 'bookImg');
+      // Book's Image
+      const img = document.createElement('img');
+      img.src = myBooks[bookTitles[i]].img;
```
[the remark this answers is not in the document]
Comment (husenjanobul) [reply Week1/homework/app.js]: Sorry, I’ll read again and do it.

**Comment by Tjebbee** [comment]
@husenjanobul ; do you still need to implement this feedback?

**Pull request closed without merging**